rvps/src/cache/local_fs/README.md

```diff
@@ -0,0 +1,15 @@
+# Local File System Cache
```

Comment:
I agree with @dubek here, `cache` is confusing.
Can we rename `src/cache` to `src/store`, as a first commit to that PR?

Reply:
Sure. I'm going to open an issue to track this problem